sopel/modules/reddit.py

```diff
@@ -113,20 +113,58 @@ def rpost_info(bot, trigger, match):
         return NOLIMIT
 
 
-# If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, commanded=False):
+    """Shows information about the given subreddit"""
+    r = praw.Reddit(
+        user_agent=USER_AGENT,
+        client_id='6EiphT6SSQq7FQ',
+        client_secret=None,
+    )
+    try:
+        r.subreddits.search_by_name(match, exact=True)
+        s = r.subreddit(match)
+        link = "https://www.reddit.com/r/" + s.display_name
+
+        tz = time.get_timezone(bot.db, bot.config, None, trigger.nick,
+                               trigger.sender)
+        time_created = dt.datetime.utcfromtimestamp(s.created_utc)
+        created = time.format_time(bot.db, bot.config, tz, trigger.nick,
+                                   trigger.sender, time_created)
+
+        message = ('[REDDIT] {link}{nsfw} | subscribers ({subscribers}) | Created at {created} | {public_description}')
+
+        nsfw = ''
+        if s.over18:
+            nsfw += ' ' + bold(color('[NSFW]', colors.RED))
+
+            sfw = bot.db.get_channel_value(trigger.sender, 'sfw')
+            if sfw:
+                link = '(link hidden)'
+                bot.kick(
+                    trigger.nick, trigger.sender,
+                    'Linking to NSFW content in a SFW channel.'
+                )
+
+        message = message.format(
+            link=link, nsfw=nsfw, subscribers=s.subscribers, created=created, public_description=s.public_description)
+        bot.say(message)
+    except prawcore.exceptions.NotFound:
```

Comment:
I've a feeling that this exception will be raised by the first line of this try/except block. So maybe you can limit it to this line, in particular.

Reply:
I'm talking about this line: `r.subreddits.search_by_name(match, exact=True)`.

Reply:
fair enough